src/client/pages/Section/hooks/useGetTableData/useGetTableData.ts

```diff
@@ -15,20 +16,21 @@ export const useGetTableData = (props: Props) => {
   const { sectionName } = props
 
   const dispatch = useAppDispatch()
-  const { assessmentName, cycleName, countryIso: _countryIso } = useCountryRouteParams()
+  const { assessmentName, cycleName, countryIso } = useCountryRouteParams<CountryIso>()
   const canEdit = useCanEdit(sectionName)
   const { print } = useIsPrintRoute()
   const dependencies = useDependencies(props)
 
-  const countryIso = _countryIso as CountryIso
+  const country = useCountry(countryIso)
+  const mergeOdp = country.props.forestCharacteristics?.useOriginalDataPoint
```

Comment:
Then we go back to previous implementation and the current commit is redundant:

```
        const withOdp =
          tableName === TableNames.extentOfForest ||
          (tableName === TableNames.forestCharacteristics && forestCharacteristicsUseOdp)
```

We need to change this to 
```
        const withOdp =
          tableName === TableNames.extentOfForest ||
          (tableName === TableNames.forestCharacteristics && forestCharacteristicsUseOdp) ||
          <pseudo: table 1a or 1b as dep>
```

But then later in the code the `originalDataPointValue` is added?

```
        if (withOdp) {
          internal.tableNames.add(TableNames.originalDataPointValue)
          internal.tableWithOdp = tableName
        }
```

Let me know

Reply:
I tried - it doesn't work if we disable ODPs from section 1b (button: 'Don't use national data points')
The problem is that now we would always fetch the ODP data (when in section 3a for example) and validate against that

Reply:
_Still is a note to keep track of investigation:_

So server must be incorrect in this case:
1. Disable ODPs from section 1b
2. Navigate to section 3a
3. You will still receive mergedODP data if mergeOdp flag is true (which it will be when we use `!tableWithOdp`